ensemble_eeg/ensemble_edf.py

```diff
@@ -373,14 +373,10 @@ def get_patient_age(header):
 
 
 def anonymize_edf_header(fd):
-    """Function to anonymize edf files according to ENSEMBLE and BIDS standards.
-    The output file will be appended with ANONYMIZED in the filename. Please
-    make sure to only upload the anonymized files.
-
-
-    Args:
-    fd (str): (Relative) path to file to rename.
+    """
+    Anonymizes an EDF file's header fields according to ENSEMBLE and BIDS standards and writes the result to a new file with '_ANONYMIZED' appended to the filename.
 
+    The function replaces patient and recording identifiers with anonymized values and recalculates the recording start date based on a fixed reference date plus the patient's age. The original data is preserved, but all identifying information in the header is removed or replaced to ensure privacy.
```

Comment:
@coderabbitai The fixed reference date 195-01-01 is often used when pseudonymising EDF files. It is not part of the EDF standard, rather it's a customary choice. It should appear as such in the documentation.

Reply:
@coderabbitai Can you modify the documentation accordingly?

Reply:
@coderabbitai Can't you update the documentation in this PR?

Reply:
I'm lost, there's too much to read and I cannot make real sense of it. Do you need anything from me?